react-native.config.js

```diff
@@ -13,3 +13,13 @@ module.exports = {
     },
   },
 };
+
+const windowsSwitch = '--use-react-native-windows';
+
+if (process.argv.includes(windowsSwitch)) {
+  process.argv = process.argv.filter(arg => arg !== windowsSwitch);
```

Comment:
I don't understand - right now I substitute the 'windowsSwitch' in argv for the '--config' option - how should I use the filtered argv? (btw, this is code pasted from our migration tutorial)

Reply:
So my comment was along the lines of: who is using process.argv and expecting to see the --config option instead of the --use-react-native-windows option? can they receive an arguments parameter and we pass them a copy of process.argv instead of modifying process.argv directly?

Reply:
Is there a pattern for solving the Metro issue used in other community modules? Was under the impression we could share metro configs (though I think things get wonky if you add macos with windows as well).